src/main.ts

```diff
@@ -439,23 +425,21 @@ ipcMain.handle(
   }
 )
 
-ipcMain.handle('har:open', async (_, fileName: string): Promise<HarFile> => {
-  console.info('har:open event received')
-  let fileHandle: FileHandle | undefined
-  try {
-    fileHandle = await open(path.join(RECORDINGS_PATH, fileName), 'r')
-    const data = await fileHandle?.readFile({ encoding: 'utf-8' })
+ipcMain.handle(
+  'har:open',
+  async (_, fileName: string): Promise<HarWithOptionalResponse> => {
+    console.info('har:open event received')
+    const data = await readFile(path.join(RECORDINGS_PATH, fileName), {
+      encoding: 'utf-8',
+      flag: 'r',
+    })
     // TODO: https://github.com/grafana/k6-studio/issues/277
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    const har = await JSON.parse(data)
+    const har = JSON.parse(data)
 
-    // TODO: https://github.com/grafana/k6-studio/issues/277
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    return { name: fileName, content: har }
-  } finally {
-    await fileHandle?.close()
+    return har
```

Comment:
small nit: can't we return directly the parse without the extra variable ?

Reply:
Done, managed to remove 2 todos along the way 😎